CMakeLists.txt

```
@@ -1,44 +1,82 @@
 cmake_minimum_required(VERSION 3.16)
+project(ramtools VERSION 1.0.0 LANGUAGES CXX)
 
-project(ramtools LANGUAGES CXX)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-find_package(ROOT 6.26 REQUIRED
-  COMPONENTS Core RIO Tree ROOTNTuple)
+option(RAMTOOLS_BUILD_TESTS "Build unit tests" ON)
+option(RAMTOOLS_BUILD_BENCHMARKS "Build benchmarks" ON)
+option(RAMTOOLS_BUILD_TOOLS "Build tools" ON)
 
-set(RNTUPLE_LIBS ROOT::ROOTNTuple ROOT::ROOTNTupleUtil)
+find_package(ROOT 6.26 REQUIRED COMPONENTS Core RIO Tree ROOTNTuple ROOTNTupleUtil)
```

Is `6.26` important. I think this code might work with any version after that one, too...

---

We should change 6.26 to 6.26 or later.

---

Still not resolved.

---

```suggestion
set(ROOT_MIN_VERSION 6.26)
find_package(ROOT ${ROOT_MIN_VERSION} REQUIRED COMPONENTS Core RIO Tree ROOTNTuple ROOTNTupleUtil)
```

Ok, then to make this behavior of cmake clear in the code...